modules/op-tkn-task-management.adoc

```diff
@@ -37,23 +38,23 @@ List tasks.
 .Example: List all the tasks in a namespace
 [source,terminal]
 ----
-$ tkn task list -n myspace
+$ tkn task list -n <namespace_name>
 ----
 
 == task logs
 Display task logs.
 
-.Example: Display logs for the `mytaskrun` task run of the `mytask` task
+.Example: Display logs for a task run of a task
```

Comment:
@eromanova97 Can you please answer this for us?

Reply:
Because we specify the task run in the command, it should show the associated task run of the task. @ppitonak is that incorrect?

Reply:
Oh, I didn't even know that this command exists :D Tasks don't have logs, task *runs* have logs. IMHO most people would run `tkn taskrun logs <task_run_name> -n <namespace_name>` which is more natural.

Reply:
In that case, there are to options how this can proceed:
* Because we already document `tkn taskrun logs` command in a taskrun section, we could remove this command
* Because this command, even though not well-known, still exists, it could stay. In that case, we should change the wording of the faulty description from `Display task logs.` to `Display logs for a task run of a task.`

I think the best option here is to remove the command, if it is not really beneficial to the user.

Reply:
@eromanova97 Okay, I have removed the command : )
Please verify the changes.

Reply:
/lgtm